packages/notification-service/package.json

```diff
@@ -24,6 +24,7 @@
     "express": "^4.17.1",
     "firebase-admin": "^8.10.0",
     "i18next": "^19.0.2",
+    "moment": "^2.29.0",
```

Comment:
Without this dependency the service fails after deploying and no notifications are sent. There might be some dependency thing I'm missing because contractkit is the one that requires it, not this service, but for some reason it's not found when deploying :(

Reply:
Oh yes, @carterqw2 bumped into the same issue recently.
What he did is update ContractKit deps: https://github.com/celo-org/celo-monorepo/pull/6700

Reply:
In my case, I solved it by upgrading @celo/contractkit and @celo/utils to 1.0.1 (moment was a missing dependency in `1.0.0-beta3`).